app/src/main/java/xyz/hexene/localvpn/UDPInput.java

```diff
@@ -73,7 +73,12 @@ public void run()
                         DatagramChannel inputChannel = (DatagramChannel) key.channel();
                         // XXX: We should handle any IOExceptions here immediately,
                         // but that probably won't happen with UDP
```

Comment:
Please see the above comment and TCPInput.java for the correct way of doing this.